AGM_Interaction/config.cpp

```diff
@@ -186,6 +187,7 @@ class AGM_Core_Options {
 
 class AGM_Parameters {
   AGM_Modifier = 0;
+  AGM_enableTeamManagement = 1;
```

Comment:
If I push him to much he'd probably laugh at me and scratch my god-awful parameter system.